source/taup/install.rst

```diff
@@ -0,0 +1,47 @@
+安装
+====
+
+1. 确认 Java 运行环境已安装
+
+在终端中键入 ``java -version``，若显示版本信息，则表示 Java 运行环境已安装::
+
+    $ java -version
+    openjdk version "1.8.0_161"
+    OpenJDK Runtime Environment (build 1.8.0_161-b14)
+    OpenJDK 64-Bit Server VM (build 25.161-b14, mixed mode)
+
+否则需要先安装 Java 运行环境::
+
+    # 在 CentOS 7 上
+    $ sudo yum install java
+
+    # 在 Ubuntu 上
+    $ sudo apt update
+    $ sudo apt install default-jre default-jdk
+
+    # 在 macOS 上
+    $ brew install openjdk
```

Comment:
Need to indent the whole block by 4 whitespaces.

Reply:
It seems error if we don't indent the block by whitespaces.

Reply:
Maybe